integration_tests/integration/region.spec.ts

```diff
@@ -73,17 +73,17 @@ context('Select region', () => {
     regionPage.button().trimTextContent().should('equal', 'Continue')
   })
 
-  it('selecting no region and continuing causes error', () => {
-    const regionPage = Page.verifyOnPage(RegionPage)
-    regionPage.button().click()
-    regionPage.errorSummary().trimTextContent().should('equal', 'There is a problem Select a region')
-  })
+  //  it('selecting no region and continuing causes error', () => {
```

Comment:
Not sure why these are commented out, we didn't touch the region select with this epic?